apps/dashboard/app/views/batch_connect/sessions/connections/_one_tab.html.erb

```diff
@@ -0,0 +1,4 @@
+<hr>
+<div class="ood-appkit markdown">
+	<%= render partial: "batch_connect/sessions/connections/#{tab[:partial]}", locals: tab[:locals] %>

```

Comment:
This file appears to be using tabs instead of spaces for whitespace. Please fix.